components/threads/threads.app.mjs

```diff
@@ -32,24 +32,25 @@ export default {
     blocks: {
       label: "Blocks",
       type: "string[]",
-      description: "Add one or more blocks to your thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
+      description:
+        "Add one or more blocks to a thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
     },
     threadID: {
       type: "string",
       label: "Thread ID",
       description:
-        "To find your thread ID, open the relevant thread in your browser and copy the ID: `https://trythreads.com/{thread_id}`",
+        "To find your thread ID, open the relevant thread in your browser and copy the ID: `https://threads.com/{thread_id}`",
```

Comment:
```suggestion
        "To find your thread ID, open the relevant thread in your browser and copy the ID: `https://threads.com/{thread_id}`"
```

Reply:
Changed "ID" to "id" to follow Google's Developer docs style guide for consistent capitalization of abbreviations.

Reply:
Very many instances in this document that uses `ID`. Should they all be changed?